src/styles/blocks/card.scss

```
@@ -0,0 +1,97 @@
+.card-cont {
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  display: flex;
+  flex-direction: column;
+
+  @include main-font-set;
+  @include main-inline-padding;
+
+  width: 200px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  &:hover {
+    @include card-hover;
+  }
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    min-height: 134px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    margin-bottom: 4px;
+  }
+
+  &__prod-code {
+    color: $secondary-accent;
+    font-size: 10px;
+    line-height: 14px; /* 140% */
```

Remove comments everywhere
```suggestion
    line-height: 14px;
```